test/test_matcher.cpp

```diff
@@ -75,6 +76,13 @@ BOOST_FIXTURE_TEST_CASE(const_char_pointer_and_std_string_can_be_compared, fixtu
     BOOST_CHECK(!match(std::string("different text"), actual));
 }
 
+BOOST_FIXTURE_TEST_CASE(null_const_char_pointers_can_be_compared, fixture)
```

Comment:
I’m not sure this even needs a fixture?

Reply:
I agree that this would indeed be simpler as a regular test case that defines `null_str` itself.